### ISSUE TITLE
[Feature request] Configurable modifier for opening URLs with left mouse button click

### ISSUE THREAD

**Issue by wasamasa** [lxde/lxterminal#95]
I was surprised that clicking links (no modifier) didn't work. Only after reading the source code, I noticed one can right click and choose the appropriate context menu item or left click while holding the control modifier key. How about a configurable mouse modifier setting to change that behavior? If there's no issues with that, I'd try to implement it myself.

**Comment by FinboySlick** [comment]
On face value it sounds like a reasonable request to me.  Is it merely a matter of usability?  

Playing devil's advocate, an argument could be made for the 'barebones but functionnal' aspect of LXTerminal that if you can achieve the same result (opening links) already, this could fall into feature-creep rather than functionality.

That being said, having more people working on LXTerminal is always good so I'd love to see your patch.

**Comment by wasamasa** [comment]
Yeah, I found it unusual that URLs are highlighted when hovering, but don't do anything on clicking. A simpler change would be to permit clicking with and without the control key.

**Comment by FinboySlick** [comment]
I believe that is by design so you don't accidentally launch a browser while clicking inside a terminal window (say to bring it in focus).

**Comment by wasamasa** [comment]
I can see the case for that. What about a boolean configurable to allow clicking without control? I'd place it under the "Advanced" tab and call it "Open URLs without control modifier" or something.

**Comment by FinboySlick** [comment]
Or maybe another row in 'Shortcuts' called something like "Link open modifier" that would default to "Ctrl".  Not sure if that's ambiguous.
